### PR TITLE
Fix circleci job mangling

### PR DESCRIPTION
**Pull request by desfero**
Tests #p3

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -15,97 +15,99 @@ executors:
...
       - run: yarn workspace @neufund/sagas lint
       - run: yarn workspace @neufund/shared-modules lint
       - run: yarn workspace @neufund/shared-utils lint
       - run: yarn workspace @neufund/web lint
+
       # Unit Tests
       - run: yarn workspace @neufund/shared-utils test
       - run: yarn workspace @neufund/web test
 
-      - save_cache:
```
Comment (desfero) [inline .circleci/config.yml:62]: this one is not true given that root package.json is never changing (at least not that often) and all `packages` node_modules are not cached so it kindy useless

File: .circleci/config.yml [hunk 2]
```diff
@@ -15,97 +15,99 @@ executors:
...
 commands:
-  notify_slack:
+  notify-slack:
     description: "Notifies slack about the job failure on master or production branches"
     steps:
       - slack/status:
           fail_only: true
           only_for_branches: master,production
 
+  checkout-and-install:
```
Comment (arjunu) [inline .circleci/config.yml:25]: 👍

File: .circleci/config.yml [hunk 3]
```diff
@@ -15,97 +15,99 @@ executors:
     working_directory: /home/circleci/app
 
 commands:
-  notify_slack:
+  notify-slack:
     description: "Notifies slack about the job failure on master or production branches"
     steps:
       - slack/status:
           fail_only: true
           only_for_branches: master,production
```
Comment (arjunu) [inline .circleci/config.yml:24]:
```suggestion
failure_message: ':red_circle: $CIRCLE_JOB failed for $CIRCLE_BRANCH!'
```

Comment (desfero) [reply .circleci/config.yml:25]: done

**Pull request merged**